site/content/integrate/reference/personal-access-token/_index.md

```diff
@@ -14,11 +14,11 @@ Personal access tokens function similar to session tokens and can be used by int
 2. Identify the account you want to create a personal access token with. You may optionally create a new user account for your integration, such as for a bot account. By default, only System Admins have permissions to create a personal access token.
 3. To create an access token with a non-admin account, you must first give it the appropriate permissions. Go to **System Console > User Management > Users**, search for the user account, then select **Manage Roles** from the dropdown.
 
-    ![image](access_token_manage_roles.png)
+    ![image](access_token_enable.png)
 
 4. Select **Allow this account to generate personal access tokens.**
 
-    ![image](access_tokens_additional_roles.png)
+    ![image](access_token_profile.png)
```

Comment:
@aniketh-varma

1. This screenshot isn't correct. Please refer to the [existing docs page](https://developers.mattermost.com/integrate/reference/personal-access-token/) as a reference.
2. Avoid renaming screenshot files. Please reuse the same filename.
3. Ensure that file extensions are lowercase (not PNG).